### PR TITLE
Add docker container

### PR DESCRIPTION
**Pull request by Terverok**
Adds configured docker container for project
Build: sudo docker build -t secret-szanta .
Run: sudo docker run -p 80:3000 secret-szanta

### CODE REVIEW

**Comment by Dammic** [comment]
Loo docker thats cool

**Comment by Dammic** [comment]
I assume you need docker for that, right? You should include more information about that (ex. installation) in the readme.
Also, please install docker on raspberry pi

**Comment by Dammic** [comment]
Need to investigate logs.

File: README.md [hunk 1]
```diff
@@ -19,9 +19,23 @@ npm run dev
 
 That will build the application files, start the watcher process and run the server.
 
+#DOCKER CONTAINER
```
Comment (Dammic) [inline README.md]: Add information about it in REadme:
Building docker and running docker should be done on the production only now due to lack of watch feature.

File: package.json [hunk 2]
```diff
@@ -50,47 +50,48 @@
...
     "image-webpack-loader": "3.4.2",
-    "merge-stream": "~1.0.1",
+    "merge-stream": "1.0.1",
     "node-sass": "4.5.3",
     "nodemon": "1.12.1",
-    "npm-run-all": "~4.0.2",
-    "postcss-loader": "~1.3.3",
+    "npm-run-all": "4.0.2",
+    "postcss-loader": "1.3.3",
+    "rimraf": "2.6.2",
```
Comment (Dammic) [inline package.json]: probably unnecessary rimraf added

Comment (Terverok) [reply package.json]: removed

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+RUN npm install
+
+COPY . .
+
+# exposing port inside of container to outside
+EXPOSE 3000
+
+RUN npm run build:dll
+RUN npm run build
```
Comment (Dammic) [inline Dockerfile]: You need to change this to npm run build:prod

**Pull request merged**